states/georgia/elections.toml

```diff
@@ -20,9 +20,8 @@ received_by = 2020-10-05
 sources = ["https://sos.ga.gov/admin/files/2020%20Revised%20Short%20Calendar.pdf"]
 
 [20201103.absentee.application]
-email_by = 2020-11-03
-in_person_by = 2020-11-03
-received_by = 2020-11-03
+email_by = 2020-10-20 # Padded 14 Days
+received_by = 2020-10-20 # Padded 14 Days
```

Comment:
I think you want fax and in_person still. "Absentee ballot applications can be returned by mail, fax, email (as an attachment), or in-person to the local County Board of Registrar’s Office."

Reply:
👍